envoy-control-core/src/main/kotlin/pl/allegro/tech/servicemesh/envoycontrol/snapshot/ServiceTagFilter.kt

```diff
@@ -1,6 +1,6 @@
 package pl.allegro.tech.servicemesh.envoycontrol.snapshot
 
-class ServiceTagFilter(properties: ServiceTagsProperties = ServiceTagsProperties()) {
+open class ServiceTagFilter(properties: ServiceTagsProperties = ServiceTagsProperties()) {
```

Comment:
why it has to be open?

Reply:
Now it doesn't. I was a leftover from when I wanted to extend this class for our use case.